docs/source/whatsnew/development.rst

```diff
@@ -170,6 +170,89 @@ For more information please see unit test -
 
 =======
 
+History Range Glob feature
+==========================
+
+Previously, when using ``%history``, users could specify either
+a range of sessions and lines, for example:
+
+.. code-block:: python
+
+   ~8/1-~6/5   # see history from the first line of 8 sessions ago,
+               # to the fifth line of 6 sessions ago.``
+
+Or users could specify a glob pattern:
+
+.. code-block:: python
+
+   -g <pattern>  # glob ALL history for the specified pattern.
```

Comment:
would it make sense to use a specific pattern here, as you did in the range example above? Then the comment could highlight what the pattern corresponds to. I don't have a strong feeling about this, but thought I'd mention it to see if that makes sense to you.

Reply:
ignore my comment above, that documentation came from a separate PR altogether.